ff/README.md

```diff
@@ -16,6 +16,7 @@ This crate contains two types of traits:
 
 The available field traits are:
 
+- [`AdditiveGroup`](/ff/src/lib.rs) - Interface for additive groups that can be generated by a `ScalarField`. This applies both to prime-order fields, field extensions, and elliptic-curve groups used in cryptography.
```

Comment:
`AdditiveGroup` only has a `Scalar` associated type, I think this needs to be updated since the naming changed

Reply:
right, good catch, fixing.